components/form/src/index.js

```diff
@@ -1,11 +1,14 @@
 import { AuroForm } from './src/auro-form.js';
+import { AuroDropdown } from './src/dropdown/auro-dropdown.js';
+import { AuroInput } from './src/input/auro-input.js';
+import { MySelect } from './src/dropdown/select.js';
 
 /**
  * Register Custom Element.
  * @param {Object} name - Name to use for custom element.
  * @returns {void}
  */
- const registerComponent = (name = 'custom-form') => {
+const registerComponent = (name = 'custom-form') => {
```

Comment:
Custom component registration config needs to be refactored.

Reply:
Will defer and track in the respective ticket to update the registration logic: https://github.com/AlaskaAirlines/auro-form/issues/21